package.json

```diff
@@ -231,25 +231,11 @@
     "caniuse-lite": "1.0.30001563",
     "@storybook/core-common": "7.6.19",
     "storybook-builder-parcel/@storybook/core-common": "7.6.19",
-    "@parcel/cache": "2.0.0-dev.1601",
-    "@parcel/codeframe": "2.0.0-dev.1601",
-    "@parcel/diagnostic": "2.0.0-dev.1601",
-    "@parcel/events": "2.0.0-dev.1601",
-    "@parcel/fs": "2.0.0-dev.1601",
-    "@parcel/logger": "2.0.0-dev.1601",
-    "@parcel/markdown-ansi": "2.0.0-dev.1601",
-    "@parcel/node-resolver-core": "3.3.1-dev.3224",
-    "@parcel/package-manager": "2.0.0-dev.1601",
-    "@parcel/plugin": "2.0.0-dev.1601",
-    "@parcel/profiler": "2.12.1-dev.3224",
-    "@parcel/rust": "2.12.1-dev.3224",
-    "@parcel/types": "2.0.0-dev.1601",
-    "@parcel/utils": "2.0.0-dev.1601",
-    "@parcel/workers": "2.0.0-dev.1601",
```

Comment:
i've confirmed that we only have one copy of each of these installed, so great that we've been able to remove them from resolutions